x/stakeibc/types/message_claim_undelegated_tokens.go

```diff
@@ -47,11 +45,6 @@ func (msg *MsgClaimUndelegatedTokens) ValidateBasic() error {
 	if err != nil {
 		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
-	// sender must be a valid stride address
```

Comment:
is there any reason to check the receiver address validity here? don't think there is (since it's verified in `RedeemStake`), but just double-checking